src/openforms/forms/utils.py

```diff
@@ -387,6 +394,11 @@ def apply_definition_conversions(configuration: JSONObject) -> None:
         converter(configuration)
 
 
+def apply_registration_backend_conversions(configuration: list[dict]) -> None:
+    for converter in REGISTRATION_BACKEND_CONVERTERS:
+        converter(configuration)
```

Comment:
Looking at the data migration which doesn't make use of this constant, I don't see a benefit in adding an abstraction for this here and would much favour explicitness in the options serializers to perform the normalization.

The `CONVERTERS` is different, because we have a custom migration operation that allows you to refer to a converter operation and you don't have to code data migrations anymore, while ensuring that the same logic runs during import/export of forms. Those conditions do not apply here, so let's simplify the implementation.